### DISCUSSION TITLE
Titles has style and would like to remove it

### DISCUSSION THREAD

**Discussion by ldmax** [nvim-neorg/neorg#1208] [Q&A] [answered]
Hi community!

Today I opened my Neorg note as usual, and found titles are with styles.
I updated yesterday.

Please see below screenshot:
![image](https://github.com/nvim-neorg/neorg/assets/4306230/40cec409-118a-4d9a-bbef-27c8a2cd69f3)

Could someone kindly tell me what these styles are? Before yesterday there are no such highlights. I would like to remove them.
Thanks!

**Comment by pysan3** [comment, marked as answer]
Do you have this plugin installed? https://github.com/lukas-reineke/headlines.nvim

What does `:Inspect` say when you put your cursor on one of the headings.

**Reply by ldmax** [reply]
Yes it's because of headlines.nvim. After adding below configuration, the style is gone:
```lua
return {
  {
    "iulas-reineke/headlines.nvim",
    config = function()
      require("headlines").setup({
        norg = {
          headline_highlights = false,
        } 
      })
    end
  }
}
```

Thanks pysan3!
